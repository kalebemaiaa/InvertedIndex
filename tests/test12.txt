QueryAndL(QueryOr(QueryWord("diplomata"), QueryAnd(QueryWord("ministro"), QueryOr(QueryWord("assessor"), QueryWord("coordenador")))), QueryOr(QueryWord("chanceler"), QueryAnd(QueryWord("embaixador"), QueryOr(QueryWord("secretário"), QueryWord("governador")))))
1435.raw
5444.raw
3636.raw
11514.raw
2398.raw
3024.raw
11493.raw
194.raw
4873.raw
4876.raw
4239.raw
3444.raw
3745.raw
4792.raw
2125.raw
11005.raw
1472.raw
2769.raw
4285.raw
11074.raw
1054.raw
1352.raw
2074.raw
900.raw
1402.raw
5536.raw
11693.raw
5512.raw
1733.raw
1987.raw
2270.raw
1156.raw
1055.raw
3524.raw
5382.raw
303.raw
11984.raw
11628.raw
2658.raw
3788.raw
5060.raw
1543.raw
3546.raw
3982.raw
6028.raw
4799.raw
5829.raw
2464.raw
11201.raw
2782.raw
6239.raw
5345.raw
10983.raw
3071.raw
10806.raw
3246.raw
8730.raw
11244.raw
6241.raw
216.raw
4108.raw
3019.raw
4091.raw
2837.raw
5103.raw
4590.raw
2934.raw
11524.raw
2841.raw
10821.raw
2246.raw
11954.raw
5793.raw
908.raw
4427.raw
3249.raw
6018.raw
4958.raw
872.raw
2520.raw
11321.raw
3075.raw
6111.raw
11456.raw
1642.raw
3068.raw
4208.raw
2807.raw
3722.raw
11250.raw
2773.raw
6303.raw
2139.raw
5373.raw
2356.raw
1480.raw
2308.raw
5623.raw
11925.raw
11233.raw
3621.raw
4539.raw
11516.raw
3773.raw
3617.raw
1273.raw
778.raw
5601.raw
2429.raw
2441.raw
5236.raw
987.raw
5922.raw
4630.raw
5894.raw
11771.raw
1589.raw
11433.raw
11762.raw
1736.raw
1299.raw
11967.raw
4308.raw
11810.raw
5483.raw
5475.raw
4704.raw
11226.raw
11996.raw
935.raw
5578.raw
450.raw
350.raw
3277.raw
5440.raw
4018.raw
5480.raw
4137.raw
3165.raw
4895.raw
8755.raw
6389.raw
4023.raw
4611.raw
4554.raw
4475.raw
2234.raw
6257.raw
4495.raw
2182.raw
11797.raw
3098.raw
11174.raw
3313.raw
3115.raw
4525.raw
2084.raw
5756.raw
1876.raw
4226.raw
11085.raw
2448.raw
3599.raw
11217.raw
214.raw
3260.raw
3188.raw
10949.raw
11090.raw
2572.raw
6016.raw
1969.raw
11236.raw
4625.raw
6191.raw
11338.raw
4403.raw
4246.raw
3596.raw
269.raw
3268.raw
4143.raw
3949.raw
958.raw
5722.raw
5322.raw
704.raw
10974.raw
3171.raw
793.raw
5856.raw
4079.raw
4678.raw
1231.raw
2221.raw
4329.raw
3363.raw
3554.raw
11796.raw
983.raw
