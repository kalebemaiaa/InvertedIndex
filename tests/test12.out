3949
4079
5440
